zanata-client-commands/src/test/java/org/zanata/client/commands/init/TransConfigPromptTest.java

```diff
@@ -109,4 +109,29 @@ public void allowUserToRetry() throws Exception {
 
         assertThat(opts.getTransDir(), Matchers.equalTo(new File("po")));
     }
+
+    @Test
+    public void canHandleFileTypeProjectAndDisplayTransFiles() throws Exception {
+        ConsoleInteractor console =
+                MockConsoleInteractor.predefineAnswers(".", "y");
+        opts.setProjectType("file");
+        opts.setProj("about-fedora");
+        opts.setProjectVersion("master");
+        // provides locale list
+        LocaleList locales = new LocaleList();
+        locales.add(new LocaleMapping("de"));
+        opts.setLocaleMapList(locales);
+        prompt =
+                new TransConfigPrompt(console, opts, Sets.newHashSet("a.txt",
+                        "b.txt"));
+
+        prompt = prompt.promptUser();
+
+        assertThat(opts.getTransDir(), Matchers.equalTo(new File(".")));
+        List<String> capturedPrompts =
+                MockConsoleInteractor.getCapturedPrompts(console);
+        assertThat(capturedPrompts, Matchers.hasItems(
+                "        ./de/a.txt",
+                "        ./de/b.txt"));
+    }
```

Comment:
Do we have any tests that look at what the user would actually see? It would be great to have tests for the console interactions something like:

```
userReads("What file types do you want to use?");
userReads("Hint: ...");
userReads("Please enter file types in comma separated format (e.g. XML_DOCUMENT_TYPE_DEFINITION,IDML[txt]):");
userTypes("PLAIN_TEXT");
userReads(...);
```

so it is easy to see what the real interaction should be like, and easy to change it with TDD.